docs/getting_started.rst

```diff
@@ -74,13 +74,12 @@ Managing Python installations can be quite tricky. We *highly* recommend using p
 Python virtual environment
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-You should use a Python virtual environment to isolate the dependencies of your Python projects from each other and to avoid corrupting your system's Python installation. There are many ways to set up Python virtual environments, including Virtualenv which has been around for quite some time. Learn more about `using Virtualenv <https://virtualenv.pypa.io/en/stable/userguide/>`__.
-
-.. note::
-  Many virtual environments will require special setup for non-Bash shells such as Fish and ZSH.
+You should use a Python virtual environment to isolate the dependencies of your Python projects from each other and to avoid corrupting your system's Python installation.
 
+There are many ways to set up Python virtual environments, but we recommend installing and using `Pipenv <https://pipenv.readthedocs.io/en/latest/>`__. (You could also use `Virtualenv <https://virtualenv.pypa.io/en/stable/userguide/>`__, `Python 3 venv <https://docs.python.org/3/library/venv.html>`__, `Poetry <https://poetry.eustace.io>`__, etc...)
```

Comment:
Pipenv makes named environments a non-obvious choice, for instance switching back and forth between `develop` and `release-v0.12.x`. I'm not sure why it should figure as a recommendation? Before, we didn't have recommendations, I just think we based the examples on pipenv at some point, making that an implicit recommendation :)

Reply:
I'm not sure if I'm correct, but to use named virtualenvs, you actually have to revert to classic tools?

> Current versions of pipenv respect an existing virtualenv, so if you want to control the name you could (with virtualenvwrapper)

`https://github.com/pypa/pipenv/issues/1226`

Reply:
```suggestion
There are many ways to set up Python virtual environments: You can use `Pipenv <https://pipenv.readthedocs.io/en/latest/>`__ as shown in the instructions below. You can also use `Virtualenv <https://virtualenv.pypa.io/en/stable/userguide/>`__, `Python 3 venv <https://docs.python.org/3/library/venv.html>`__, `Poetry <https://poetry.eustace.io>`__, etc.
```

Reply:
committed suggestion. 

> Pipenv makes named environments a non-obvious choice, for instance switching back and forth between develop and release-v0.12.x

I don't actually understand what you mean by this.

Reply:
If I'm developing on the branch `release-v0.12.x`, then the set of installed dependencies is different and I'd want a different virtualenv than say developing on `develop`.

I normally use virtualenvwrapper, i.e. "workon <name>" and "mkvirtualenv -p <python2|python3> <name>" quite often to switch back and forth between Kolibri and Python versions. This can be refined and optimized, the point is just to save a bit of time and space on (re)creating virtual environments.

If I understand pipenv correctly, it only bases the environment name on the path of the repository? So changing branches will mean you have to run `pipenv install -r requirements.txt` ?

Reply:
Minor note, I run `pip install -r requirements.txt`, not `pipenv` once the the virtual environment is active.

Switching python versions is indeed a bit annoying. Switching branches on the other hand is generally quite fast.